### PR TITLE
PES-2296, PES-2348 Check phpcs sniff ValidVariableName

### PR DESCRIPTION
**Pull request by mzk**

### CODE REVIEW

File: src/Packetery/Module/Checkout.php [hunk 1]
```diff
@@ -1413,8 +1413,8 @@ public function filterPaymentGateways( array $availableGateways ): array {
 		}
 
 		$order = null;
-		if ( isset( $wp->query_vars['order-pay'] ) && is_numeric( $wp->query_vars['order-pay'] ) ) {
-			$order = $this->orderRepository->getById( (int) $wp->query_vars['order-pay'], true );
+		if ( isset( $wp->query_vars['order-pay'] ) && is_numeric( $wp->query_vars['order-pay'] ) ) { // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
+			$order = $this->orderRepository->getById( (int) $wp->query_vars['order-pay'], true ); // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```
Comment (Karpec) [inline src/Packetery/Module/Checkout.php:1417]: proč tady to ignorujeme? Takhle to nikdo nikdy neopraví. nejde to přesunou t do adapteru?

File: src/Packetery/Module/Upgrade.php [hunk 2]
```diff
@@ -181,8 +181,8 @@ public function check(): void {
 		}
 
 		if ( $oldVersion && version_compare( $oldVersion, '1.4.2', '<' ) ) {
-			$version_1_4_2 = new Version_1_4_2( $this->wpdbAdapter );
-			$version_1_4_2->run();
+			$version142 = new Version_1_4_2( $this->wpdbAdapter );
```
[the remark this answers is not in the document]
Comment (mzk) [reply src/Packetery/Module/Upgrade.php:184]: updated.

File: src/Packetery/Module/Options/Exporter.php [hunk 3]
```diff
@@ -237,7 +237,7 @@ private function formatVariable( $variable, int $level = 0, bool $addSeparator =
 		} elseif ( $variable instanceof \WC_Shipping_Method ) {
 			$methodInfo = [
 				'id'           => $variable->id,
-				'method_title' => $variable->method_title,
+				'method_title' => $variable->method_title, // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```
[the remark this answers is not in the document]
Comment (mzk) [reply src/Packetery/Module/Options/Exporter.php:240]: I prefer to keep it as is

File: src/Packetery/Module/Checkout.php (hunk 1 above)
Comment (mzk) [reply src/Packetery/Module/Checkout.php:1417]: updated

File: src/Packetery/Module/Options/Exporter.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (jan-potuznik-z) [reply src/Packetery/Module/Options/Exporter.php:240]: What's the reason?

[the remark this answers is not in the document]
Comment (mzk) [reply src/Packetery/Module/Options/Exporter.php:240]: It's my personal preference. There is no reason.
The readability is the same for me.

**Review by zemekoule — APPROVED** [review]

**Pull request merged**